### DISCUSSION TITLE
Timeline for release of calendar aria & stately packages?

### DISCUSSION THREAD

**Discussion by kevinmpowell** [adobe/react-spectrum#1915] [General]
Any idea when https://github.com/adobe/react-spectrum/tree/1e2b7f280a04ead83e21b27fa580eb189e5f4186/packages/%40react-aria/calendar

and https://github.com/adobe/react-spectrum/tree/1e2b7f280a04ead83e21b27fa580eb189e5f4186/packages/%40react-stately/calendar

alpha's will be released? I have an existing styled datepicker component that needs a11y enhancements and this looks like it'd be a good fit.

**Comment by dannify** [comment]
Hey @kevinmpowell, we don't have an exact date for DatePicker to be released as an alpha but we will be doing some work on it in the coming month or so.
